templates/evaluations.html

```diff
@@ -8,16 +8,20 @@ <h2>Evaluation form</h2>
         <p>Please note that your answers are confidential and that if there are any problems or changes we have to
             report, we'll do so anonymously.</p>
         <hr>
-        <form action="{{ url_for("course.create_evaluation", current_year=current_year, user_id=user_id) }}"
+        <form action="{{ url_for("course.create_evaluation", current_year=current_year, user_id=session["user_id"]) }}"
```

Comment:
Do we want to be able to create evaluations for another user ? If not, sepcifying the user id in the route is useless and bug-prone.